src/baserow/client.py

```diff
@@ -98,6 +98,27 @@ def token_refresh(self, token: str) -> t.Tuple[User, str]:
     response = self._request('POST', '/api/user/token-refresh/', json=payload).json()
     return databind.json.load(response['user'], User), response['token']
 
+  def create_user(
+    self,
+    name: str,
+    email: str,
+    password: str,
+    authenticate: t.Optional[bool] = False,
+    group_invitation_token: t.Optional[str] = None,
+    template_id: t.Optional[int] = None
+  ) -> t.Tuple[User, str]:
```

Comment:
```suggestion
  ) -> t.Tuple[User, t.Optional[str]]:
```

If you want to be even fancier, you could specify an overload when `authenticate` is set to `True` (i.e. a token will be returned) or `False` (no token will be returned). But I don't see that as a necessity now. 😄